AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -393,6 +393,7 @@ describe('AddParticipantComponent', () => {
             expect(city.value).toBe(participant.city);
             expect(county.value).toBe(participant.county);
             expect(postcode.value).toBe(participant.postcode);
+            expect(false).toBe(participant.isRepresenting);
```

Comment:
shouldn't this be written as `expect(participant.isRepresenting).toBeFalsy();`

Reply:
Done